lib/FusionInventory/Agent/Task/Inventory/MacOS/Hostname.pm

```diff
@@ -1,29 +1,37 @@
 package FusionInventory::Agent::Task::Inventory::MacOS::Hostname;
 
+use English qw(-no_match_vars);
+
 use strict;
 use warnings;
 
+use Sys::Hostname;
+
 use FusionInventory::Agent::Tools;
 use FusionInventory::Agent::Tools::MacOS;
 
 sub isEnabled {
-    return canRun('/usr/sbin/system_profiler');
+    return 1 if $OSNAME eq 'darwin';
```

Comment:
This is useless here: anything under Inventory::MacOS namespace wll only be run if the condition of Inventory::MacOS::isEnabled function is already satisfied.